statusCodes.js

```diff
@@ -0,0 +1,20 @@
+var NullValue = {
+  status: 403, 
+  message:'Sorry, you have passed undefined'
+};
+
+var ErrorString = {
+  status: 404,
+  message:'Sorry, the number is greater than 100'
+};
+
+var ErrorLargeNumber = {
+  status: 400, 
+  message: `Sorry, hello is not an integer`
+};
+
+module.exports = {
```

Comment:
a cleaner way to do this file would be to just create and export the object at the same time. e.g.:

```
module.exports = {
  foo: {
    bar: 'baz'
  }, 
  bar: {
    baz: 'foo'
  }
}
```

Reply:
it saves you some extra variable assignments, and is just as readable